portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,82 @@
 
 package com.google.sps.servlets;
 
+import java.lang.StringBuffer;
 import java.io.IOException;
+import java.time.format.DateTimeFormatter;  
+import java.time.LocalDateTime;   
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private ArrayList<Comment> allComments = new ArrayList<Comment>();
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+    /**
+    * Class to organize comments for more detailed displays. By creating 
+    * another class, adding new parameters and information to keep track 
+    * of may be easier.
+    */
+    private static class Comment {
```

Comment:
I like the usage of a Comment class - something to consider, you are currently storing the date as a String. How will that work across different time zones, daylight savings time, etc, etc. Is there a better way?

Reply:
If I leave a comment at +6 GMT, how will that look to someone in -2 GMT for example?

Reply:
One of the more standard ways to store time, is as milliseconds (or seconds) since Jan 1 1970 GMT - this is referred to as "epoch" time. This has a couple advantages, 1) you can use a long or int, which is much easier to reason about or deal with than classes like ZonedDateTime 2) it's well understood by programmers everywhere, in all languages 3) you -always- want to store time in a time zone agnostic way, which means using GMT. For time, remember that pretty much the only time you want to bring time zones into the picture is when you actually have to display times to the user. That means that your backend should basically never deal with timezones, leave that up to your front end.

For more interesting reading, see: https://infiniteundo.com/post/25326999628/falsehoods-programmers-believe-about-time

Reply:
I decided to just leave the date/time out for now. It's a feature that I like, but I do not want to think about it anymore right now when I have other things to do. 
:)